crates/astria-core/src/sequencer/v1/block/mod.rs

```diff
@@ -14,8 +14,8 @@ use super::{
     IncorrectRollupIdLength,
     RollupId,
 };
-use crate::{
-    sequencer::v1::{
+use crate::sequencer::{
+    v1::{
```

Comment:
nit: unnecessary change, kinda surprised format doesn't dislike it

Reply:
what do you mean? this has to be changed because of the protobuf trait change